content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
Oh, I see what you're talking about.  I'm not sure that swapping in "block" fixes this.  I think we would have the same potential ambiguity with the wait operations and potentially also other collectives that block (e.g. for non-root PEs in broadcast).  If we think this is an ambiguity that needs to be fixed, something at a higher level might be needed.